packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/attachcomponentsetc.md

```diff
@@ -1,18 +1,112 @@
 ## Attach Components to the Secure Router
 
-You need to provide these routes in your sample application so that we can sign the user in and handle the callback from Okta. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
+We provide the `Security` component which makes an Auth object available to child components and the `withAuth` HOC. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
```

Comment:
We provide the `Security` component that makes an Auth object available to child components and the `withAuth` HOC. We show you how to set these up using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):